doc/index.rst

```diff
@@ -90,6 +89,27 @@ Learning resources
           - Subplots (`.pyplot.subplots`, `.pyplot.subplot_mosaic`)
 
 
+************
+Live example
+************
+
+Try Matplotlib directly in this documentation (Press ``shift+Enter`` to execute code)! Alternatively, you can try the gallery examples in `our JupyterLite deployment <./lite/lab>`__.
+
+.. replite::
+   :kernel: xeus-python
+   :height: 600px
+   :prompt: Try Matplotlib!
+
+   %matplotlib inline
```

Comment:
```suggestion
   %matplotlib widget
```

Reply:
If we are going to embed an interactive section we should use a live backend!

Reply:
I totally agree, although widgets don't work in the replite UI unfortunately.

Alternatively we could show the retrolab UI instead, which does support widgets. But it's a more "crowded" UI. What do you think?

Reply:
I would go for an interactive backend over almost anything.

Reply:
Sounds good! I will push the require changes